Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM docker.io/postgres:15.2
+
+COPY ./databaseSetup/db-setup.sql /docker-entrypoint-initdb.d/
+COPY ./databaseSetup/db-setup-testdata.sql /docker-entrypoint-initdb.d/
```

Comment:
Why 2 separate copy lines, didn't you have it copying the whole database setup directory?